blocks/results-list-block/features/results-list/results/mock-data.js

```diff
@@ -0,0 +1,296 @@
+export default {
+	_id: "55FCWHR6SRCQ3OIJJKWPWUGTBM",
+	type: "story",
+	version: "0.10.4",
+	content_elements: [
+		{
+			_id: "L6RMSXTS7RB5XFKZPPAZKXGRF4",
+			type: "text",
+			additional_properties: [Object],
+			content:
+				"No, August isn’t the District’s hottest month, not quite. It’s actually July. But as the area is in the thick of yet another heat wave, August surely feels like it.",
+		},
+		{
+			_id: "BQ3NJWT6UBDCJNDONKJW7E3I2E",
+			type: "text",
+			additional_properties: [Object],
+			content: "Here’s why such feelings can be deceptive.",
+		},
+		{
+			_id: "QYR6VILSJBEJNI45LOBXYIUWNA",
+			type: "text",
+			additional_properties: [Object],
+			content:
+				"The numbers plainly show that July’s heat reigns supreme. The table below displays 30-year “normal” or average temperatures for July and August at Washington’s Reagan National Airport (DCA) from 1981 to 2010 as well as the records from 1871 to the present.",
+		},
+		{
+			_id: "3J4YJHWBLVEXXEQW5O2724TTVY",
+			type: "table",
+			header: [Array],
```

Comment:
Same here with `[Array]`.  If these values are not used in the test, we should remove them even if it makes the object incomplete.  If we aren't using the data, there's no need for it.